common/StackCanary.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include "common/Types.h"
+
+asm void StackCanary_Init(void);
+asm u32 StackCanary_Canary(void);
```

Comment:
Can the `asm` part be moved to the c file?

Reply:
Resolved in commit [7716b67](https://github.com/doubledashdeluxe/ddd/pull/66/commits/7716b677b06aafacbcc399c725836f8cdc6ee1dc).